resources/js/features/games/components/HashesMainRoot/HashesMainRoot.tsx

```diff
@@ -75,9 +96,78 @@ export const HashesMainRoot: FC = memo(() => {
             />
           </p>
 
-          <HashesList />
+          <HashesList hashes={hashes} />
+        </div>
+
+        {hasOtherHashes ? (
+          <BaseCollapsible open={isOpen} onOpenChange={setIsOpen}>
```

Comment:
nit: It may be worth considering extracting this into a child component -- it's taking up quite a bit of the HashesMainRoot template.

Reply:
done